### ISSUE TITLE
option to exclude a path

### ISSUE THREAD

**Issue by zhurkin** [HynekPetrak/log4shell-finder#2]
add arguments, option to exclude a path

**Comment by HynekPetrak** [comment]
Will add both, option to exclude path as well as to skip mount points

```
[2021-12-20 11:12:17,750] [INFO] [I] Parameters: ./test_log4shell.py / --exclude-dirs /mnt --same-fs
[2021-12-20 11:12:17,758] [INFO] [I] Analyzing paths (could take a long time).
[2021-12-20 11:12:17,759] [INFO] [I] Skipping mount point: /
[2021-12-20 11:12:17,759] [INFO] [I] Skipping mount point: /run
[2021-12-20 11:12:17,760] [INFO] [I] Skipping mount point: /run/user/1000
[2021-12-20 11:12:17,767] [INFO] [I] Skipping mount point: /run/lock
[2021-12-20 11:12:17,829] [INFO] [I] Skipping blaclisted folder: /mnt
[2021-12-20 11:12:17,830] [INFO] [I] Skipping mount point: /dev
[2021-12-20 11:12:17,830] [INFO] [I] Skipping mount point: /dev/mqueue
[2021-12-20 11:12:17,831] [INFO] [I] Skipping mount point: /dev/hugepages
[2021-12-20 11:12:17,831] [INFO] [I] Skipping mount point: /dev/shm
[2021-12-20 11:12:17,834] [INFO] [I] Skipping mount point: /dev/pts
```

**Comment by HynekPetrak** [comment]
fixed in 1.4
